src/main/java/nl/tudelft/scrumbledore/userinterface/GameDisplay.java

```diff
@@ -363,7 +357,9 @@ private static void renderPlayer() {
         if (player.getSpeed().getX() == 0 && !isFiring) {
           steps = 0;
         }
-        String path = sprites.getAnimated("player-" + color + "-" + spr).getFrame(steps).getPath();
+        String path = sprites
+            .getAnimated("player-" + Constants.PLAYER_COLORS[player.getPlayerNumber()] + "-" + spr)
```

Comment:
Nice solution for now. I think in the future we should make LevelElements or some elements containing them aware of how to draw them, instead of determining this in the GUI.
<review